recipes/windninja/meta.yaml

```diff
@@ -0,0 +1,79 @@
+# Note: there are many handy hints in comments in this example -- remove them when you've finalized your recipe
```

Comment:
Please remove these comments as stated in the comment

Reply:
fixed